website/content/docs/concepts/security/data-encryption.mdx

```diff
@@ -105,6 +105,13 @@ $ boundary scopes list-key-version-destruction-jobs -scope-id p_A4jfDjZ9jf
 Once the job disappears from this list, the associated key version will have
 been destroyed and any existing data will have been re-encrypted.
 
+## The `previous-root` KMS key <sup>OSS Only</sup>
+
+The `previous-root` KMS key is used when migrating to a new `root` key. Adding
+it to your configuration will inform the Controller to use it for decrypting
```

Comment:
```suggestion
the `previous-root` KMS key to your configuration informs the Controller to use it for decrypting
```

Reply:
Maybe repeat `previous-root` KMS key here just to make it abundantly clear what the "it" you're referring to is?